hipify_cli.py

```diff
@@ -9,23 +9,22 @@
 import os
 import sys
 import argparse
+import json
 from hipify import hipify_python
 
 parser = argparse.ArgumentParser(
     description='Top-level script for HIPifying, filling in most common parameters')
 parser.add_argument(
     '--project-directory',
     type=str,
-    default=os.getcwd(),
     help="The root of the project. (default: %(default)s)",
-    required=False)
+    required=True)
 
 parser.add_argument(
     '--output-directory',
     type=str,
-    default=os.getcwd(),
     help="The Directory to Store the Hipified Project",
-    required=False)
+    required=True)
```

Comment:
@pruthvistony We should make the output directory default to the project directory and make it required=False

Reply:
done